pthread-makefile-examples/1_pthread_create/1_thread_create.c

```diff
@@ -0,0 +1,49 @@
+/*
+* Brief - A simple example that demonstrates the creation a pthread.
+*
+*/
+
+#include <pthread.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <unistd.h>
+
+//Thread function should have return type (void *) and should receive arguments of type (void *)
+static void * threadFunc(void *arg)
+{
+
+    char *str = (char *) arg;
+
+    //On execution, note that that PID of this thread is the same as that of pthread
+    printf("ThreadFunc: arguments passed to thread are: %s , pid = (%d)\n", str,getpid());
```

Comment:
@DeekshithPatil I'd recommend a message like "Sleeping 2 seconds from inside threadfunc" around sleep statements like this.